src/manager/index.js

```diff
@@ -22,7 +22,7 @@ const ICONS_FALLBACK = {
   bitcoin_testnet: "bitcoin",
 };
 
-const oldAppsInstallDisabled = ["ZenCash", "Ripple", "Ontology"];
```

Comment:
we must keep them!

Reply:
i removed them since this function was not used on the live before and only on the test app. i figured it wasn't making sense to disable install of ripple app per instance :shrug:

Reply:
no, we must put them back.

- ZenCash is an old app that was renamed to Horizen
- Ripple is an old app that was renamed to XRP
- Ontology, same, it is ONT now.

we must keep this because we must not allow users to actually install these old apps.
the regression is probably on LLD side but we must keep this imo.